test/expected/build/application.i/dest/manifest.json

```diff
@@ -1,13 +1 @@
-{
-    "_version": "1.1.0",
-    "sap.app": {
-        "_version": "1.1.0",
-        "id": "application.i",
-        "type": "application",
-        "applicationVersion": {
-            "version": "1.2.2"
-        },
-        "embeds": ["embedded"],
-        "title": "{{title}}"
-    }
-}
+{"_version":"1.1.0","sap.app":{"_version":"1.1.0","id":"application.i","type":"application","applicationVersion":{"version":"1.2.2"},"embeds":["embedded"],"title":"{{title}}"},"sap.ui5":{"dependencies":{"libs":{"sap.ui.layout":{},"sap.ui.core":{},"sap.m":{},"sap.ui.fl":{}}}}}
```

Comment:
I don't think that this is expected. When modifying the manifest IMO, it should still be formatted, preferably with the same indentation as before, but I think tabs should be fine.